eng/tools/TypeSpecValidation/src/TypeSpecValidation.ts

```diff
@@ -1,5 +1,5 @@
 import {exec} from "child_process";
-import fs from "fs";
+import {promises as fsp} from "fs"
```

Comment:
```suggestion
import {access} from "fs/promises"
```

nicer to just import what you need if you can

Reply:
seems like it can't find the module fs/promises, looking into it

Reply:
might need to add `@types/node` as a devDependency